queries.sql

```diff
@@ -24,4 +24,85 @@ SELECT * FROM animals WHERE neutered = true
 SELECT * FROM animals WHERE name != 'Gabumon'
 
 -- Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg)
-SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
\ No newline at end of file
+SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
+
+-- Inside a Transaction
```

Comment:
- [ ] Kindly, create the queries to update the `animals` table by setting the species column to unspecified, then roll back the change. Take a look on the code snippet below:
```sql
BEGIN; -- start transaction
   UPDATE animals
   SET species = 'unspecified';
   SELECT species from animals; 
   ROLLBACK;
   SELECT species from animals;
```